packages/target-tauri/src-tauri/src/webxdc/commands_main_window.rs

```diff
@@ -356,6 +347,20 @@ pub(crate) async fn open_webxdc<'a>(
     })
     .on_navigation(move |url| url.origin_no_opaque() == initial_url_clone.origin_no_opaque());
 
+    // This is only for non-macOS platforms.
+    // Use a non-working proxy to almost(!) isolate the app
+    // from the internet.
+    // "Almost" because there are still cases where the webview
+    // will bypass the proxy, such as with WebRTC.
+    // To disable WebRTC, we take separate measures.
+    //
+    // Note that `additional_browser_args` might make `proxy_url`
+    // have no effect (see below).
```

Comment:
```suggestion
    // have no effect (see below).
    //
    // IDEA: we disabled it on macOS, because it increased the minimum version to 14.
    // But there may be a possibility to enable it conditionally when the api is available, see
    // https://github.com/deltachat/deltachat-desktop/issues/5201
```